### PR TITLE
[[ SB Inclusions ]] Update standalone builder GUI with new inclusions pane

### PR DESCRIPTION
**Pull request by livecodeali**
Standalone Settings GUI diffs: https://gist.github.com/livecodeali/72ca5d37026a365a09bc011088f718c0

New Inclusions pane:
screen shot 2016-05-10 at 11 03 25

General pane before:
screen shot 2016-05-10 at 11 01 48

General pane after:
screen shot 2016-05-10 at 11 00 37

Copy Files pane before:
screen shot 2016-05-10 at 11 01 58

Copy Files pane after:
screen shot 2016-05-10 at 11 01 55

iOS pane before:
screen shot 2016-05-10 at 11 03 43

iOS pane after:
screen shot 2016-05-10 at 11 03 36

Android pane before:
screen shot 2016-05-10 at 11 03 52

Android pane after:
screen shot 2016-05-10 at 11 03 48

### CODE REVIEW

**Comment by livecodeali** [comment]
Requires https://github.com/livecode/livecode/pull/3999

**Comment by livecodepanos** [comment]
Just a note regarding the new "Android" pane. The "In App Purchasing" section should expand if you tick the checkbox. And if you select "Samsung", it expands even more. So we have to make sure that after the expansion no content is chopped off.

**Comment by livecodeali** [comment]
Thanks @livecodepanos, well spotted. Here's an improved version
screen shot 2016-05-11 at 14 18 49

File: notes/feature-standalone_inclusions_ui.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Standalone Inclusions Interface
+
+The standalone settings user interface has been reworked to unify the 
+notion of app inclusion. There is now an 'Inlcusions' pane which allows
```
Comment (peter-b) [inline notes/feature-standalone_inclusions_ui.md]: "Inlcusions" -> "Inclusions"

**Comment by peter-b** [comment]
@livecode-vulcan review ok 134e866

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 134e866fb9f064cd4e2aa5b5deb29524c579afec

**Pull request merged**